source/fe/fe_bdm.cc

```diff
@@ -266,94 +282,102 @@ FE_BDM<dim>::get_ria_vector (const unsigned int deg)
 }
 
 
+namespace
+{
+  // This function sets up the values of the polynomials we want to
+  // take moments with in the quadrature points. In fact, we multiply
+  // thos by the weights, such that the sum of function values and
+  // test_values over quadrature points yields the interpolated degree
+  // of freedom.
+  template <int dim>
+  void
+  initialize_test_values (std::vector<std::vector<double> > &test_values,
+                          const Quadrature<dim> &quadrature,
+                          const unsigned int deg)
```

Comment:
No. I just got annoyed again about the output arguments in the middle of the argument list in FETools.